app/coffee/DocArchiveManager.coffee

```diff
@@ -11,7 +11,7 @@ thirtySeconds = 30 * 1000
 module.exports = DocArchive =
 
 	archiveAllDocs: (project_id, callback = (err, docs) ->) ->
-		MongoManager.getProjectsDocs project_id, (err, docs) ->
+		MongoManager.getProjectsDocs project_id, true, (err, docs) ->
```

Comment:
would an opts = {include_deleted:bool} be more explicit. Not necessarily, just thinking.